feature-proposals.rst

```diff
@@ -30,111 +36,175 @@ a new Wiki page containing an RFC with a unique title MUST be created. In this
-of the RFC text (i.e. to the actual proposal, excluding future scope, rejected
-features and references) the cooldown period MUST be reset. The cooldown period
-MUST be reset to 2 weeks (336 hours) in case of major changes. It MUST be reset
-to 1 week (168 hours) in case of minor changes if the current remaining cooldown
-period is shorter. When in doubt, the change MUST be treated as a major change.
 Major and minor changes MUST be announced in the official discussion thread,
 either in a dedicated email summarizing a list of changes or in a reply to
 another email that clearly indicates that changes to the RFC text have been made
-in response to that email.
-
-As an example: An RFC is announced at 2025-02-01 15:00. The initial cooldown
-period ends at 2025-02-15 15:00. On 2025-02-03 16:00 a minor change to the RFC
-text is made. The cooldown period is not reset, since the remaining cooldown
-period of ~12 days is longer than 1 week. On 2025-02-05 17:00 a major change to
-the RFC text is made. The cooldown period is reset to 2025-02-19 17:00. On
-2025-02-15 16:00 another minor change is made. The cooldown period is reset to
-2025-02-22 16:00. On 2025-02-21 13:00 an obvious typo is fixed. The cooldown
-period is not reset.
-
-Major changes to the RFC text include any changes that would lead to a change in
-the implementation, particularly any changes to the proposed semantics or
-syntax, updating the API stub, adding, changing or removing any voting widget.
-Minor changes to the RFC text include adding new examples, updating existing
-examples, adding additional explanation or clarification, or any other changes
-that are not purely editorial.
-
-The discussion phase MAY be extended beyond the required cooldown period. It
-SHOULD appropriately be extended in holiday periods or in cases of significant
-activity on the mailing list to allow everyone to catch up with the discussion.
-
-After a period of 6 weeks without any email in the official discussion thread,
-the discussion is considered inactive and MUST be restarted with a cooldown
-period of 1 week (168 hours) before the RFC may proceed to a vote, since it is
-likely that other RFC discussions are at the top of mind of the mailing list
-participants. After a period of 3 months without any email in the official
-discussion thread, the discussion MUST be restarted with a cooldown period of 2
-weeks (336 hours).
-
-This does not preclude discussion on the merits on any idea or proposal on the
-list without formally submitting it as a proposal, but the cooldown period is
-measured only since the formal discussion announcement as described above.
+in response to that email. If a series of changes are included together, the
+whole announcement belongs to the highest level of any of the involved changes.
+The initial proposal of the RFC is considered a Major change announcement.
+
+Cooldown period
+===============
+
+Both Major and Minor change announcements trigger a "cooldown period" to allow
```

Comment:
Should we also capitalize `Cooldown Period` to make it more clear that it is a “special term”?